helm/coco-kafka-bridge/app-configs/kafka-bridge_publishing_devcj_eu.yaml

```diff
@@ -0,0 +1,81 @@
+# Default values that will be applied to the prod cluster.
+# In delivery we have bridges only to the pub cluster of the prod env
+
+bridges:
+## Start staging bridges
+- name: cms-kafka-bridge-pub-staging-eu
+  sourceKafkaProxyUrl: "https://upp-staging-publish-eu.ft.com"
+  groupIdPrefix: kafka-bridge-pub-staging-eu
+  topic: NativeCmsPublicationEvents
+  producer: "http://cms-notifier:8080"
+  authSecretName: pub-auth
+  authSecretKey: pub-staging-auth
+  type: "plainHTTP"
+  replicas: 2
+- name: cms-kafka-bridge-pub-staging-us
+  sourceKafkaProxyUrl: "https://upp-staging-publish-us.ft.com"
+  groupIdPrefix: kafka-bridge-pub-staging-us
+  topic: NativeCmsPublicationEvents
+  producer: "http://cms-notifier:8080"
+  authSecretName: pub-auth
+  authSecretKey: pub-staging-auth
+  type: "plainHTTP"
+  replicas: 2
+#CMS metadata
+- name: cms-metadata-kafka-bridge-pub-staging-eu
+  sourceKafkaProxyUrl: "https://upp-staging-publish-eu.ft.com"
+  groupIdPrefix: metadata-kafka-bridge-pub-staging-eu
+  topic: NativeCmsMetadataPublicationEvents
+  producer: "http://cms-metadata-notifier:8080"
+  authSecretName: pub-auth
+  authSecretKey: pub-staging-auth
+  type: "plainHTTP"
+  replicas: 2
+- name: cms-metadata-kafka-bridge-pub-staging-us
+  sourceKafkaProxyUrl: "https://upp-staging-publish-us.ft.com"
+  groupIdPrefix: metadata-kafka-bridge-pub-staging-us
+  topic: NativeCmsMetadataPublicationEvents
+  producer: "http://cms-metadata-notifier:8080"
+  authSecretName: pub-auth
+  authSecretKey: pub-staging-auth
+  type: "plainHTTP"
+  replicas: 2
+## END staging bridges
+## Begin prod bridges
+#CMS
+- name: cms-kafka-bridge-pub-prod-eu
+  sourceKafkaProxyUrl: "https://upp-prod-publish-eu.ft.com"
+  groupIdPrefix: kafka-bridge-pub-prod-eu
+  topic: NativeCmsPublicationEvents
+  producer: "http://cms-notifier:8080"
+  authSecretName: pub-auth
+  authSecretKey: pub-prod-auth
+  type: "plainHTTP"
+  replicas: 2
+- name: cms-kafka-bridge-pub-prod-us
+  sourceKafkaProxyUrl: "https://upp-prod-publish-us.ft.com"
+  groupIdPrefix: kafka-bridge-pub-prod-us
+  topic: NativeCmsPublicationEvents
+  producer: "http://cms-notifier:8080"
+  authSecretName: pub-auth
+  authSecretKey: pub-prod-auth
+  type: "plainHTTP"
+  replicas: 2
+#CMS metadata
+- name: cms-metadata-kafka-bridge-pub-prod-eu
```

Comment:
These bridges should also post to cms-metadata-notifiers, so that we can have a full valid annotations publish flow in the team environment.

Reply:
Done